apps/client/src/routes/dashboards/dashboard/dashboard-page.tsx

```diff
@@ -32,10 +32,9 @@ export function DashboardPage() {
         ...dashboardQuery.data?.definition,
         // TODO: remove display settings once dynanic sizing is released
         displaySettings: {
-          numRows: 200,
+          numRows: 600,
```

Comment:
How does this work?

Reply:
it makes the dashboard longer, 3 times longer

Reply:
This number affects the height of the grid, how many cells in height it should be. Previously, the grid was setup as a square (200x200). This would mean that if the center pane size has a vertical rectangle aspect ratio (like an phone screen) the grid would not span the entire height of the pane. This is bad because the gestures only work on the grid, but the dotted background isn't visible enough for you to see that you are trying to click / drag outside of it.